cli/main.c

```diff
@@ -1143,35 +1143,38 @@ static int print_fw_part_info(struct switchtec_dev *dev)
 	int nr_mult = 16;
 	struct switchtec_fw_image_info act_img, inact_img, act_cfg, inact_cfg,
 		mult_cfg[nr_mult];
-	struct switchtec_fw_footer map;
-	struct switchtec_fw_footer bootloader;
-	char map_ver[16];
+	struct switchtec_fw_image_info bl2_act_img, bl2_inact_img;
+	struct fwdl_meta bootloader;
 	char bootloader_ver[16];
 	int bootloader_ro;
+	unsigned long partition_start;
+	size_t partition_length;
 	int ret, i;
 
+	/*fetch fw image info from GAS */
 	ret = switchtec_fw_img_info(dev, &act_img, &inact_img);
 	if (ret < 0)
 		return ret;
 
-	ret = switchtec_fw_cfg_info(dev, &act_cfg, &inact_cfg, mult_cfg,
-				    &nr_mult);
-
+	/*fetch cfg file info from GAS */
```

Comment:
I'd suggest we can keep these three lines as is, if there's no break of coding convention.

Reply:
just remove comments.